src/_css/AllocationsComponent.css

```diff
@@ -0,0 +1,192 @@
+@font-face {
```

Comment:
We are supposed to be using sass now. Please change the file to `.scss`

Reply:
This is the generated file. I must have pushed by mistake.